pyproject.toml

```diff
@@ -1,7 +1,12 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
 [ProjectConfig]
 # Only path variables can and must contain 'DIR', 'PATH' or 'ROOT' in their name as
 # these tokens are used to identify path variables from other variable types.
 PROJECT_ROOT = "."
+PROJECT_NAME = "Mbed devices"
```

Comment:
Why is this required? Would I be a terrible nitpicker if I asked for it to be consistent with `README.md` where we call this `Mbed Devices` (both uppercase)?

Reply:
My mistake.
```suggestion
PROJECT_NAME = "Mbed Devices"
```